src/Uplink/Admin/Plugins_Page.php

```diff
@@ -32,7 +32,7 @@ class Plugins_Page {
 	 *
 	 * @return void
 	 */
-	public function display_plugin_messages( string $page ): void {
```

Comment:
@borkweb Matt, these type check are causing fatals when trying to move tickets as $page is null. 

I removed them, but I could also do a check in the provider instead and leave the the type check.

Reply:
@jesseeproductions sorry, this is my fault for trusting the existing docblocks when they were clearly wrong. I think type juggling in the provider is a great idea, however if you want to leave it this way, please update the `@param` dockblock to show the correct type coming in: `string|null`

Reply:
@jesseeproductions mind if I take a stab at just moving the type juggling into the provider?

Reply:
No Worries @defunctl I updated the docblocks. 

My preference is to leave it this way and not have logic in the Provider

Reply:
All good, I did add this which also fixes the tests/PHPStan issues (separate from this): https://github.com/stellarwp/uplink/pull/56